Framework/Core/Windows/Devices/Enumeration/DeviceInformation.cs

```diff
@@ -0,0 +1,98 @@
+using System;
+
+namespace Windows.Devices.Enumeration
+{
+    public sealed class DeviceInformation
+    {
+        // Construction and destruction

```

Comment:
Loose the blatantly obvious comments such as "constructor" etc... they don't actually provide any information that anyone knowing the language couldn't determine on their own with ease. Do, however, add Full XML doc comments on all public and protected language elements. While we don't have that for all our existing code, we should endeavor to ensure all new libraries include it.